### DISCUSSION TITLE
Clickable/Linkable Ticks

### DISCUSSION THREAD

**Discussion by errantmind** [f5/unovis#219] [Q&A] [answered]
Hello, I have a stacked horizontal bar chart showing some metrics for different 'sites', which are along the y-axis (the ticks). I'd like to make those sites clickable to go to particular pages, or maybe trigger a custom 'on click' event. Is this possible?

**Comment by rokotyan** [comment, marked as answer]
@errantmind You can set up events for Axis components just like for any other component. Here's how to do it for a click event:

```tsx
import { VisAxis,  ... } from '@unovis/react'
import { Axis } from '@unovis/ts'

...
<VisAxis
  ...
  events={{
    [Axis.selectors.tick]: {
      click: (tickValue: number) => console.log(tickValue),
    },
  }}
/>
```

**Reply by errantmind** [reply]
Great, thanks

**Reply by rokotyan** [reply]
@errantmind You're welcome!

We'll soon release a new version that will also allow additional styling of the tick labels: https://github.com/f5/unovis/pull/223

**Reply by rokotyan** [reply]
@errantmind Unovis 1.2 has just been [released](https://github.com/f5/unovis/releases/tag/1.2.0) with new CSS variable to style your axis:
<img width="1197" alt="image" src="https://github.com/f5/unovis/assets/755708/ff4243f5-0fbe-42e9-abc8-f402dfd9ce4a">

**Reply by errantmind** [reply]
Very nice, thanks
